.actions/assistant.py

```diff
@@ -73,25 +37,74 @@ def _retrieve_files(directory: str, *ext: str) -> List[str]:
     return all_files
 
 
-class AssistantCLI:
-    _PATH_ROOT = str(Path(__file__).parent.parent)
-    _PATH_SRC = os.path.join(_PATH_ROOT, "src")
+def _replace_imports(lines: List[str], mapping: List[Tuple[str, str]]) -> List[str]:
+    """Replace imports of standalone package to lightning.
+
+    >>> lns = [
+    ...     "lightning_app",
+    ...     "delete_cloud_lightning_apps",
+    ...     "from lightning_app import",
+    ...     "lightning_apps = []",
+    ...     "lightning_app and pytorch_lightning are ours",
+    ...     "def _lightning_app():",
+    ...     ":class:`~lightning_app.core.flow.LightningFlow`"
+    ... ]
+    >>> mapping = [("lightning_app", "lightning.app"), ("pytorch_lightning", "lightning.pytorch")]
+    >>> _replace_imports(lns, mapping)  # doctest: +NORMALIZE_WHITESPACE
```

Comment:
it is easier to read with pprint

Reply:
We can add backlashes if you prefer, but the current break conforms to the line limit. pprint is not necessary